.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,76 @@
+name: "\U0001F680 Feature Request"
+description: Suggest an idea for this project
+title: "(short issue description)"
+labels: [feature-request, needs-triage]
+body: 
+  - type: markdown 
+    attributes: 
+      value: |
+        ## Community Note
+        1. Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+        2. Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
+        3. If you are interested in working on this issue, please leave a comment
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the feature
+      description: A clear and concise description of the feature you are proposing.
+    validations:
+      required: true
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your Feature Request related to a problem?
```

Comment:
capitalization of "Feature Request" seems out of place and inconsistent with other sections

Reply:
Good point, it's not a title or proper name. Will fix.